bigtable/admin_test.go

```diff
@@ -25,6 +26,83 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
+type mockTableAdminClock struct {
+	btapb.BigtableTableAdminClient
+
+	getTableReq    *btapb.GetTableRequest
+	updateTableReq *btapb.UpdateTableRequest
+	getTableResp   *btapb.Table
+}
+
+func (c *mockTableAdminClock) GetTable(
```

Comment:
Is this function ever used?

Reply:
If it isn't used (right now), it's probably best to send an UNIMPLEMENTED response to the client. That helps future maintainers to know that the logic isn't complete.